### PR TITLE
Docs: Self-Hosted Section + Updated Quickstart Guide

### PR DESCRIPTION
**Pull request by erondpowell**
TODO:
- [x] Create Self Hosted Section
- [x] Move Installation and Localhost Quickstart Guide into Self Hosted
- [x] Re-edit first draft of `getting-started > quickstart` for the Cloud
- [x] Update Media
- [x] Request Manager Review + Comments
- [x] Apply Manager's Suggested Edits
- [x] address this: https://discordapp.com/channels/725371605378924594/915654933590720573/935222846425288724
- [x] Final Proofread

### CODE REVIEW

File: docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -1,170 +1,142 @@
...
+3. Now you're ready to [create a Community Project](/cloud/projects/#create-a-community-project).
 
-- **Database Host** – IP address for your database.
-- **Port** – Port number your database is running on.
-- **Database Name** – Name of your existing database.
-- **Database User** – Name of existing user in database.
-- **Database Password** – Password to enter database.
-- **Enable SSL** – Select `Y` for yes or `N` for no.
-- **Root** – Provide the root name.
+_It should take around 90 seconds for the Cloud Project to build out. During this time, a link will be sent to the email
```
Comment (lmaupas) [inline docs/getting-started/quickstart.md:19]: g

**Comment by erondpowell** [comment]
@rijkvanzanten @benhaynes It's time to take a look at this bad boi.

**Comment by erondpowell** [comment]
@benhaynes
Another thing to note: on the quickstart guide, I would like to add a conclusion.
It will point out several right tracks users can follow to keep learning.
- bite-size tutorials to learn how to build directus from scratch
- bite-size tutorials starting with demo data
- crashcourse
- contact us for enterprise info
- community
- etc.

**Comment by erondpowell** [comment]
@rijkvanzanten I'm not 100% sure why this is throwing a merge conflict. `quickstart` was simply edited and `plesk` was moved to self-hosted with no edits (along with 11 other self-hosted start guides which don't appear here).

I haven't seen this before and I am not 100% how to merge the conflict properly. Any tips??

**Comment by erondpowell** [comment]
@rijkvanzanten Do you mind "reviewing" this branch so it can get merged?

**Comment by erondpowell** [comment]
@rijkvanzanten You mind "reviewing" this or assign me as a reviewer so I can merge it?

File: docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -1,173 +1,185 @@
...
+[Directus Cloud](https://directus.cloud/login)
 
-```bash
-Create your first admin user:
-? Email: admin@example.com
-? Password: ********
-```
+Your Directus Cloud Account allows you to create and manage any number of Community and Standard Projects. We've made
+life easier by giving you the option to create and log in to your free Cloud Account automatically with Github. If you
+don't have a Github account or prefer not to use this login method, email-and-password login is available as well.
```
Comment (rijkvanzanten) [inline docs/getting-started/quickstart.md:18]:
```suggestion
life easier by giving you the option to create and log in to your free Cloud Account automatically with GitHub. If you
don't have a GitHub account or prefer not to use this login method, email-and-password login is available as well.
```

File: docs/getting-started/quickstart.md [hunk 3]
```diff
@@ -1,173 +1,185 @@
...
+Once logged in, you're greeted with the option to create your first
+[Collection](/getting-started/glossary/#collections).
 
-Follow the prompts and create a Collection. For the sake of this demo, we'll be calling ours `articles`, but feel free
-to make it your own!
+1. Navigate into the Content Module.
+2. Click **"Create Collection"** and a side menu will appear.
+3. Fill in a **Name**.\
+   _For the sake of this demo, we'll be calling ours `articles`, but feel free to make it your own!_
+4. Leave the other settings at default and click <span mi btn>arrow*forward</span> and the **"Optional System Fields"**
```
Comment (rijkvanzanten) [inline docs/getting-started/quickstart.md:86]:
```suggestion
4. Leave the other settings at default and click <span mi btn>arrow_forward</span> and the **"Optional System Fields"**
```

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Couple tiny tweaks. Other than that, LGTM!

**Comment by erondpowell** [comment]
> Couple tiny tweaks. Other than that, LGTM!

Ok. It's good. I'm still blocked from hitting the big red button.

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**